src/fides/api/ctl/database/crud.py

```diff
@@ -12,23 +12,22 @@
 from sqlalchemy import update as _update
 from sqlalchemy.dialects.postgresql import Insert as _insert
 from sqlalchemy.exc import SQLAlchemyError
+from sqlalchemy.ext.asyncio import AsyncSession
 from sqlalchemy.future import select
 
-from fides.api.ctl.database.session import async_session
 from fides.api.ctl.utils import errors
 
 
 # CRUD Functions
 async def create_resource(
-    sql_model: Base,
-    resource_dict: Dict,
+    sql_model: Base, resource_dict: Dict, async_session: AsyncSession
```

Comment:
Adjusted these crud methods to require an `async_session` as an argument which will be test-mode aware, and talk to the correct db in the correct context.